### PR TITLE
Feature/ts more

### PR DESCRIPTION
**Pull request by rikbrowning**
- [x] @featherds/checkbox
- [x] @featherds/chips
- [x] @featherds/date-input
- [x] @featherds/dialog
- [x] @featherds/drawer
- [x] @featherds/dropdown

### CODE REVIEW

File: packages/@featherds/chips/src/components/FeatherChipList.vue [hunk 1]
```diff
@@ -51,20 +52,25 @@ export default {
...
+        ),
         single,
       };
     }
-    if (format === "grid" || format === "single") {
-      const attrs = {
-        role: "grid",
-      };
-      const keydown = () => {};
+    // if (format === "grid" || format === "single") {
```
Comment (jmillar-nanthealth) [inline packages/@featherds/chips/src/components/FeatherChipList.vue]: Unneeded comment

File: packages/@featherds/checkbox/demos/Checkbox.vue [hunk 2]
```diff
@@ -33,5 +34,5 @@ export default {
   components: {
```
Comment (jmillar-nanthealth) [inline packages/@featherds/checkbox/demos/Checkbox.vue:34]: I know it wasn't added here, but can we convert the alert to be a console.log instead if it's to demonstrate something, as it's much less intrusive?

File: packages/@featherds/checkbox/demos/CheckboxGroup.vue [hunk 3]
```diff
@@ -30,9 +30,10 @@
     </FeatherCheckboxGroup>
```
Comment (jmillar-nanthealth) [inline packages/@featherds/checkbox/demos/CheckboxGroup.vue:30]: Also wasn't added in this change, but can we correct the 'Approvde' typo?

**Review by jmillar-nanthealth — CHANGES_REQUESTED** [review]
Address the comments and this is golden

File: packages/@featherds/chips/src/components/FeatherChipList.vue (hunk 1 above)
Comment (rikbrowning) [reply packages/@featherds/chips/src/components/FeatherChipList.vue]: So i left this commend in to document what the else roles are, I'll reword it a bit better

File: packages/@featherds/checkbox/demos/CheckboxGroup.vue (hunk 3 above)
Comment (rikbrowning) [reply packages/@featherds/checkbox/demos/CheckboxGroup.vue:30]: fixed

File: packages/@featherds/checkbox/demos/Checkbox.vue (hunk 2 above)
Comment (rikbrowning) [reply packages/@featherds/checkbox/demos/Checkbox.vue:34]: fixed

**Review by jmillar-nanthealth — APPROVED** [review]

**Pull request merged**